docs/conf.py

```diff
@@ -0,0 +1,64 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+import sphinx_rtd_theme
+sys.path.insert(0, os.path.abspath('.'))
+from squad.version import __version__
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'SQUAD'
+copyright = '2020, Linaro'
+author = 'Linaro'
+version = __version__
+release = __version__
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['sphinx.ext.autodoc',
+              'sphinx.ext.coverage',
+              'sphinx.ext.viewcode',
+              'sphinx_rtd_theme',
+              'recommonmark']
+
+source_suffix = ['.txt', '.md']
+
+master_doc = 'index'
+copyright = '2016-2020, Linaro Limited'
```

Comment:
it's repeated from line 24. This one is correct.

Reply:
fixed